src/native/libs/build-native.proj

```diff
@@ -20,6 +20,9 @@
     <_CMakeArgs Condition="'$(WasmEnableThreads)' == 'true'">$(_CMakeArgs) -cmakeargs "-DCMAKE_USE_PTHREADS=1"</_CMakeArgs>
     <_CMakeArgs Condition="'$(_IcuDir)' != ''">$(_CMakeArgs) -cmakeargs "-DCMAKE_ICU_DIR=$(_IcuDir)"</_CMakeArgs>
     <_CMakeArgs Condition="'$(_TzdDir)' != ''">$(_CMakeArgs) -cmakeargs "-DCMAKE_TZD_DIR=$(_TzdDir)"</_CMakeArgs>
+    <_CMakeArgs>$(_CMakeArgs) -cmakeargs "-DCMAKE_NET_CORE_APP_CURRENT_VERSION=$(NetCoreAppCurrentVersion)"</_CMakeArgs>
```

Comment:
It would be nice to add e. g. `eng/native.targets` that would have these in an item group which we could then reuse here and in `build-runtime.proj` (there are other things that could be reused but no need to change them here obviously).

Reply:
done